writer_test.go

```diff
@@ -550,6 +550,44 @@ func TestNewMasterPlaylistWithAlternatives(t *testing.T) {
 	if m.ver != 4 {
 		t.Fatalf("Expected version 4, actual, %d", m.ver)
 	}
+	fmt.Printf("%v\n", m)
+}
+
+// Create new master playlist supporting closed-caption=none
+func TestNewMasterPlaylistWithClosedCaptionEqNone(t *testing.T) {
+	m := NewMasterPlaylist()
+	var alts = []*Alternative{
```

Comment:
One more thing, is this still required? If it's just to test for captions == none, I don't think we need an alternative for it? I'm not familiar with alternative or captions myself in m3u8 spec, so if an Alternative is required to set Captions to none then I'm OK with leaving it in, but if it's not required, then I think we can remove this from here too.